src/xcode/ENA/ENA/Source/Scenes/ENSetting/ExposureNotificationSettingViewController.swift

```diff
@@ -317,30 +317,11 @@ final class ExposureNotificationSettingViewController: UITableViewController, Ac
 	}
 
 	private func tracingCell(for indexPath: IndexPath, in tableView: UITableView) -> UITableViewCell {
-		guard let tracingCell = tableView.dequeueReusableCell(withIdentifier: ReusableCellIdentifier.tracingCell.rawValue, for: indexPath) as? TracingHistoryTableViewCell else {
+		guard let tracingCell = tableView.dequeueReusableCell(withIdentifier: ReusableCellIdentifier.daysSinceInstallationCell.rawValue, for: indexPath) as? DaysSinceInstallTableViewCell else {
```

Comment:
Please rename tracingCell

Reply:
Please provide a suggestion for a new naming. In my opinion "tracing" is still ok. The text is about tracing. Please keep in mind, this PR does not remove tracing. It only removes the logging of the tracing being active or not.

Reply:
> > I would request to run the snapshots based UI tests in ENAUITests_01_Home to see if where they are working properly. These changes will affect those tests. These snapshots tests are not a part of CI checks for this PR.
> 
> I think because we had a discussion recently and decided, that Screenshot-Tests should not block us from merging PRs, they are not part of the CI checks.
> Nearly every UI-Change will affect the Screenshots. So why blocking this PR particular?

Good point, however, these changes will directly affect the snapshot UI tests, so I will propose to remove them because they are not valid any more and add new relevant ones :)

Reply:
daysSinceInstallationCell